CONTRIBUTING.md

```diff
@@ -140,6 +140,8 @@ Start the development server.
   npm start
   ```
 
+> Now you can point your web browser to https://localhost:5601 and start using Kibana! When running `npm start`, Kibana will also log that it is listening on port 5603 due to the base path proxy, but you should still access Kibana on port 5601.
```

Comment:
Another thing (sorry if I missed this in the previous review): I think this paragraph should go *after* the "> On Windows, ..." paragraph, and it should not include a `>` so it appears as normal text.  So it would look like this:

![screen shot 2016-12-23 at 12 51 54 pm](https://cloud.githubusercontent.com/assets/53814/21459706/a2a90e4a-c90e-11e6-98cc-2bc7d81ba2c3.png)

This way the "on windows" thing is still explaining a needs for running the previous command, whereas the new text is simply explaining what's next.